roles/readarr/defaults/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+# enable or disable the application
+readarr_enabled: false
+readarr_available_externally: false
+
+# directories
+readarr_data_directory: "{{ docker_home }}/readarr"
+readarr_download_directory: "{{ downloads_root }}"
+
+# uid / gid
+readarr_user_id: "998"
```

Comment:
Where did this uid / gid come from?

Reply:
Good catch. 998 is my ansible-nas user. I'll have to go test and figure out why I choose that long ago. Probably should be 1000 if 0 doesn't work.

Reply:
updated to use root like all the other *arr linuxserver app defaults